src/Common/src/Interop/FreeBSD/Interop.Libraries.cs

```diff
@@ -5,6 +5,12 @@ internal static partial class Interop
 {
     private static partial class Libraries
     {
-        internal const string LibRt = "librt"; // POSIX Realtime Extensions library
+        /// <summary>
+        /// We aren't OS X so don't have an INODE64 suffix to entry points
+        /// </summary>
+        internal const string INODE64SUFFIX = "";
+
+        internal const string LibRt = "librt";  // POSIX Realtime Extensions library
+        internal const string libc = "libc";    // C runtime
```

Comment:
@sokket Thanks for your implementation, but still there is no "BSD". There are at least 4 major platforms with BSD in their name and we should not make the mistake to assume they all behave the same. Please drop that idea of a common BSD before this one get merged. Thanks!